### DISCUSSION TITLE
Hash based routing

### DISCUSSION THREAD

**Discussion by pranavxc** [nuxt/nuxt#16783] [Questions] [answered]
How can I switch to hash-based routing in Nuxt3? I tried following in `nuxt.config.ts` but it hasn't worked.

```js
export default defineNuxtConfig({

  // other config

  router: {
    mode: 'hash',
  },

}
```

**Comment by sevillaarvin** [comment, marked as answer]
Try `router.options.hashMode` https://v3.nuxtjs.org/guide/directory-structure/pages#hash-mode-spa

**Reply by TheAlexLichter** [reply]
Exactly! Implemented via https://github.com/nuxt/framework/pull/6980
